src/csvcubed/utils/sparql_handler/column_component_info.py

```diff
@@ -0,0 +1,22 @@
+"""
+ColumnComponentInfo
+-------------------
+
+Provides access to mapping between CSV columns.
+"""
+
+from dataclasses import dataclass
+from typing import Optional
+
+from csvcubed.models.sparqlresults import ColumnDefinition, QubeComponentResult
+from csvcubed.utils.qb.components import EndUserColumnType
+
+
+@dataclass
+class ColumnComponentInfo:
+    """This class holds information for mapping between CSV columns"""
+
+    component_type: EndUserColumnType
+    component: Optional[QubeComponentResult]
```

Comment:
It might be helpful to provide an example of the circumstances in which the component isn't defined here.